source/includes/aggregation/multi-field-join.js

```diff
@@ -0,0 +1,160 @@
+const { MongoClient } = require("mongodb");
+
+const uri = "<connection string>";
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    const aggDB = client.db("agg_tutorials_db");
+
+    // start-colls
+    const collName1 = await aggDB.collection("products");
+    const collName2 = await aggDB.collection("orders");
+    // end-colls
+
+    const sampleData1 = [
+      // start-products
+      {
+        name: "Asus Laptop",
+        variation: "Ultra HD",
+        category: "ELECTRONICS",
+        description: "Great for watching movies",
+      },
+      {
+        name: "Asus Laptop",
+        variation: "Standard Display",
+        category: "ELECTRONICS",
+        description: "Good value laptop for students",
+      },
+      {
+        name: "The Day Of The Triffids",
+        variation: "1st Edition",
+        category: "BOOKS",
+        description: "Classic post-apocalyptic novel",
+      },
+      {
+        name: "The Day Of The Triffids",
+        variation: "2nd Edition",
+        category: "BOOKS",
+        description: "Classic post-apocalyptic novel",
+      },
+      {
+        name: "Morphy Richards Food Mixer",
+        variation: "Deluxe",
+        category: "KITCHENWARE",
+        description: "Luxury mixer turning good cakes into great",
+      },
+      // end-products
+    ];
+
+    const sampleData2 = [
+      // start-orders
+      {
+        customer_id: "elise_smith@myemail.com",
```

Comment:
S: The style guide recommends using "example.com" for emails (applies to the other emails too)

```suggestion
        customer_id: "elise_smith@example.com",
```

Reply:
Ah okay you can ignore this suggestion then!